getting-started/multiband-analysis.rst

```diff
@@ -145,10 +146,23 @@ Then make an index array from the combination of ``detect_isPatchInner`` and ``d
 
    inInnerRegions = refTable['detect_isPatchInner'] & refTable['detect_isTractInner']
 
+Rejecting sky objects
+--------------------------
```

Comment:
Ensure that the headline's underline strictly matches the length of the headline.